controllers/memberstatus/memberstatus_controller_test.go

```diff
@@ -325,6 +325,25 @@ func TestOverallStatusCondition(t *testing.T) {
 			HasRoutes("https://console.member-cluster/console/", "https://codeready-codeready-workspaces-operator.member-cluster/che/", routesAvailable())
 	})
 
+	t.Run("when missing only one NodeMetrics resource then it's fine", func(t *testing.T) {
+		// given
+		memberOperatorDeployment := newMemberDeploymentWithConditions(status.DeploymentAvailableCondition(), status.DeploymentProgressingCondition())
+
+		unraedyNodeAndMetrics := newNodesAndNodeMetrics(forNode("worker-unready", []string{"worker"}, "3000000Ki"))
+		reconciler, req, fakeClient := prepareReconcile(t, defaultMemberStatusName, newGetHostClusterReady, allNamespacesCl, append(nodeAndMetrics, unraedyNodeAndMetrics[0], memberOperatorDeployment, newMemberStatus())...)
```

Comment:
```suggestion
		reconciler, req, fakeClient := prepareReconcile(t, defaultMemberStatusName, newGetHostClusterReady, allNamespacesCl, append(nodeAndMetrics, unreadyNodeAndMetrics[0], memberOperatorDeployment, newMemberStatus())...)
```

Reply:
also, why do you need to use `append()`? Can't you just do something like 
```
reconciler, req, fakeClient := prepareReconcile(t, defaultMemberStatusName, newGetHostClusterReady, allNamespacesCl, nodeAndMetrics, unreadyNodeAndMetrics[0], memberOperatorDeployment, newMemberStatus())
```
as in the next test?

Reply:
also, could you add comment to explain why the node is "unready"? It's not clear to me at first sight, because I'm not familiar with the logic, so a comment would be helpful.

Reply:
No, I cannot do that, because the `nodeAndMetrics` is  an array and the `prepareReconcile` is a variadic function

Reply:
ok, thanks, I missed that point :/